remix/prisma/schema.prisma

```diff
@@ -11,13 +11,28 @@ datasource db {
 }
 
 model User {
-  id        String   @id @default(uuid())
-  name      String   @db.VarChar(100)
-  email     String   @db.VarChar(50)
-  avatarUrl String?  @map("avatar_url")
-  createdAt DateTime @default(now()) @map("created_at")
+  id         String       @id @default(uuid())
+  name       String       @db.VarChar(100)
+  email      String       @db.VarChar(50)
+  avatarUrl  String?      @map("avatar_url")
+  createdAt  DateTime     @default(now()) @map("created_at")
+  updateAt   DateTime     @updatedAt
+  Newsletter Newsletter[]
 
   @@unique([email])
   @@index([email])
   @@map("users")
 }
+
+model Newsletter {
+  id       String   @id @default(uuid())
+  name     String   @db.VarChar(255)
+  content  String   @db.Text()
+  user     User     @relation(fields: [userId], references: [id])
+  userId   String   @map("user_id")
+  createAt DateTime @default(now()) @map("created_at")
+  updateAt DateTime @updatedAt
```

Comment:
Just a minor but I think you can also add the `@map("updated_at")` so that our columns can all be formatted into the standard snake case.

Reply:
Oh, I actually miss that, Thanks for mentioning me that 🙏🏼.

Reply:
Add Here: [c029d3e](https://github.com/nimblehq/js-framework-benchmark/pull/112/commits/c029d3edb4cc15f60d193699885fa7e27c3a895f)